spec/views/users/show.html.erb_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+RSpec.describe 'User show', type: :feature do
+  describe 'As a Visitor' do
+    describe 'I am sent to user show page ' do
+      before :each do
+        @user = User.create(name: 'Jackie Chan', email: 'a@a.com', password: 'a', password_confirmation: 'a')
+        @post = Post.create(
+          content: "MyText",
+          grass_tags: "Grass Tags",
+          only_followers: true
+        )
+
+        UserPost.create(user_id: @user.id, post_id: @post.id)
+      end
+
+      it "visits user page" do
+        visit user_path(@user)
+
+        expect(page).to have_content('Name')
+        expect(page).to have_content('Jackie Chan')
+        expect(page).to have_content('Email')
+        expect(page).to have_content('a@a.com')
+        expect(page).to_not have_link('Edit')
+        expect(page).to_not have_link('Destroy')
+        expect(page).to have_link('Back')
+      end
+
+      it 'clicks on name on main page' do
+        allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(@user)
+
+        visit user_path(@user)
+
+        expect(page).to have_content('Name')
+        expect(page).to have_content('Jackie Chan')
+        expect(page).to have_content('Email')
+        expect(page).to have_content('a@a.com')
+        expect(page).to have_link('Edit')
+        expect(page).to have_link('Destroy')
+        expect(page).to have_link('Back')
+      end
+    end
+  end
+end
```

Comment:
Ant this was the last test I had to edit to make work with how things are being presented on the page.  The tests for `:views` wasn't recognizing the `current_user` method, and erroring out, so I had to change it to `:feature` tests.  I should maybe move these tests into the `/features` folder, but I like having them next to the ones that are titled similarly and do similar things.  I could retitle them too, but still, I like this for now.